node/index.js

```diff
@@ -98,7 +98,9 @@ module.exports = options => {
 		next();
 	});
 
-	let hasher = str => str;
+	let hasher = {
```

Comment:
Would you mind deleting https://github.com/Financial-Times/n-ui/blob/0caf851bce8265fbf12b00160c7d8b050398d944/node/index.js#L115.

Also I think it should be exposed on the app as just a function with a more descriptive name... `getHashedAssetUrl`?
(most of the code in the assets bits and pieces I imagine will be refactored massively as it's a mess, so it shouldn't really inform what the public api looks like)